pkg/reconciler/broker/broker_test.go

```diff
@@ -1026,30 +1023,12 @@ func createChannelCRD(namespace string, t channelType, ready bool) *unstructured
 	}
 }
 
-func createTriggerChannelRef() *corev1.ObjectReference {
-	return &corev1.ObjectReference{
-		APIVersion: "eventing.knative.dev/v1alpha1",
-		Kind:       "Channel",
-		Namespace:  testNS,
-		Name:       fmt.Sprintf("%s-kn-trigger", brokerName),
-	}
-}
-
-func createIngressChannelRef() *corev1.ObjectReference {
-	return &corev1.ObjectReference{
-		APIVersion: "eventing.knative.dev/v1alpha1",
-		Kind:       "Channel",
-		Namespace:  testNS,
-		Name:       fmt.Sprintf("%s-kn-ingress", brokerName),
-	}
-}
-
 func createTriggerChannelCRDRef() *corev1.ObjectReference {
```

Comment:
It would also be great to get rid of the CRD in the method name.

Reply:
good point! :-)